nft_ingester/src/program_transformers/bubblegum/db.rs

```diff
@@ -75,6 +90,12 @@ where
         txn.execute(query)
             .await
             .map_err(|db_err| IngesterError::StorageWriteError(db_err.to_string()))?;
+
+
+        // Insert the audit item after the insert into cl_items have been completed
```

Comment:
Wouldn't you want insert to the audit table _before_ inserting to `cl_items`, so that if the insert to `cl_items` fails, it remains in the audit table and is audited and fixed later?